agent.go

```diff
@@ -156,6 +159,7 @@ var callbackList = map[hyper.HyperCmd]cmdCb{
 	hyper.ReadyCmd:           readyCb,
 	hyper.PingCmd:            pingCb,
 	hyper.WinsizeCmd:         winsizeCb,
+	hyper.PsContainerCmd:     processListCb,
```

Comment:
Because it has to return a reply, it should be part of a different map. Create a new one called `callbackWithReplyList`